### PR TITLE
client channel top up invoker

### PR DESCRIPTION
**Pull request by furkhat**

### CODE REVIEW

File: ui/export_test.go [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package ui
+
+import "github.com/privatix/dappctrl/job"
+
+func (h *Handler) SetMockQueue(q job.Queue) {
```
Comment (ababo) [inline ui/export_test.go]: Do we really need this function? If you insist please rename with `test.go` with a corresponding build tag.

File: ui/errors.go [hunk 2]
```diff
@@ -10,16 +10,20 @@ const (
...
-	ErrAccessDenied:     "access denied",
-	ErrInternal:         "internal server error",
-	ErrAccountNotFound:  "account not found",
-	ErrOfferingNotFound: "offering not found",
-	ErrBadObjectType:    "bad object type",
+	ErrAccessDenied:            "access denied",
+	ErrInternal:                "internal server error",
+	ErrAccountNotFound:         "account not found",
+	ErrChannelNotFound:         "channel not found",
+	ErrDefailtGasPriceNotFound: "default gas price settings not found",
```
Comment (ababo) [inline ui/errors.go]: Typo

File: ui/handler_test.go [hunk 3]
```diff
@@ -1,4 +1,4 @@
-package ui
+package ui_test
```
Comment (ababo) [inline ui/handler_test.go]: Why different package?

Comment (furkhat) [reply ui/handler_test.go]: Isolation of tests. I think it is generally better.

File: ui/export_test.go (hunk 1 above)
Comment (furkhat) [reply ui/export_test.go]: It is a conventional way to export something for tests. Filename is also a convention.

Comment (ababo) [reply ui/export_test.go]: Didn't know about that option, agree, that's better.

Comment (ababo) [reply ui/export_test.go]: Can you move other ui/ tests to the test package as well?

Comment (furkhat) [reply ui/export_test.go]: I already did that. But only this package.

**Comment by furkhat** [comment]
@simon updated.

**Pull request merged**